c/tskit/tables.c

```diff
@@ -7000,6 +7051,45 @@ tsk_table_collection_read_format_data(tsk_table_collection_t *self, kastore_t *s
     }
     memcpy(self->file_uuid, uuid, TSK_UUID_SIZE);
     self->file_uuid[TSK_UUID_SIZE] = '\0';
+
+    ret = kastore_containss(store, "metadata");
```

Comment:
Good catch, this was potentially masking errors. I'd say using ``if (ret < 0) `` is *slightly* more idiomatic for detecting the error condition, but it's not worth holding the PR up for.

Reply:
Agreed, should be that.